cds_ils/migrator/api.py

```diff
@@ -325,12 +368,12 @@ def import_serial_from_file(dump_file, rectype):
                         json_record["title"], rectype
                     )
                 )
-                has_children = json_record.get("_migration", {}).get(
-                    "children", []
-                )
-                if has_children:
-                    record = import_record(json_record, model, provider)
-                    records.append(record)
+            has_children = json_record.get("_migration", {}).get(
+                "children", []
+            )
+            if has_children:
```

Comment:
if it does not have children, do we skip the import?

Reply:
this code was moved here https://github.com/CERNDocumentServer/cds-ils/blob/44d95c114b76394e70971b0c680e13d2161cf555/cds_ils/migrator/series/series_import.py#L106 
I think this `if` shouldn't be there but maybe @kprzerwa knows better.

Reply:
this if was introduced to avoid creating the record of serial type with no documents in it, to avoid empty serials